app/controllers/close_contacts_controller.rb

```diff
@@ -54,4 +54,22 @@ def update
                                created_by: current_user.email,
                                comment: "User edited a close contact (ID: #{cc.id}).")
   end
+
+  # Delete an existing close contact record
+  def destroy
+    redirect_to root_url && return unless current_user.can_create_patient_close_contacts?
+    cc = current_user.get_patient(params.permit(:patient_id)[:patient_id])&.close_contacts&.find_by(id: params.permit(:id)[:id])
+    redirect_to root_url && return if cc.nil?
+    cc.destroy
+    if cc.destroyed?
+      reason = params.permit(:delete_reason)[:delete_reason]
```

Comment:
I'm seeing blank delete reasons in the history comment

Reply:
@tstrass see my above comment, it's not being passed as a param